inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/region_yolo_gpu_ref.cl

```diff
@@ -12,94 +12,79 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "include/common.cl"
-#include "include/data_types.cl"
-
-#define IW INPUT0_SIZES[0]
-#define IH INPUT0_SIZES[1]
-#define IC INPUT0_SIZES[2]
-#define IB INPUT0_SIZES[3]
+#include "include/include_all.cl"
```

Comment:
Could you please replace this include_all with really used include files as it impacts binary size?

Reply:
Yes, I'll update it.